README.md

```diff
@@ -1,2 +1,22 @@
-# Selenium-testNg-Maven
-This Repository contains boilerplate template for Selenium Test Automation framework.
+# Selenium-Java_BoilerCode
+
+# Introduction
+This Test Automation Framework is created using Java + Selenium Web Driver + TestNG+ Maven Page Object Model(POM). Which can be used across different web based applications.It is used to make the code more readable, maintainable, and reusable.
+
+# Pre-requisite:
+ 1. Java<br/>
+ 2. Maven<br/>
+ 3. Browsers (Firefox, Chrome)<br/>
+
+ 
+# Steps to clone execute the tests
+ `git clone https://github.com/Shiwani01/Selenium-Java_BoilerCode.git` <br/>
+ `cd Selenium-Java_BoilerCode`<br/>
+ `mvn clean test`
+ 
+# Screenshot:
+* Take Screenshots On test failures screenshots will automatically be taken and stored under build/screenshots. The screenshot files will be named of the test       method name.
```

Comment:
@Shiwani01 Take Screenshots On test failures Method will automatically capture & store the screenshots under `/Screenshots` directory.